### ISSUE TITLE
provide explanation for decision categories

### ISSUE THREAD

**Issue by herbertblb** [erictapen/mapping-migration#5]
We need to provide information on the categories "rejected", "Recognized" etc to explain what exactly they mean. Maybe we can do this via infoboxes next to the categories.

**Comment by herbertblb** [comment]
add that "total" means only total decisions about applicants of one country in one year

**Comment by seragarez** [comment]
Examine "complementary protection" more closely: What types of protection does that include? Examine relation to subsidiary and temporary protection?

**Comment by erictapen** [comment]
I could use an explanation :) @herbertblb @seragarez @yellowkobra 

In my head I'm currently trying to merge the [categories of the API](https://api.unhcr.org/docs/refugee-statistics.html#api-Default-asylumDecisions) and those in the current sketch by @Findus2000.

The API gives us the following categories:

![tmp 83fBtnA0kn](https://user-images.githubusercontent.com/11532355/149182887-4d0c31c7-0a00-438e-a2d8-7a9973098379.png)

While in our sketch we use recognized, complementary protection, rejected and other.

Am I right about this mapping?

- `dec_recognized` -> recognized
- `dec_other` -> complementary protection
- `dec_rejected` -> rejected
- `dec_closed` -> other

**Comment by seragarez** [comment]
Yes that's correct! But I would call the last one (dec_closed) "otherwise closed" instead of "other" in our vis.

**Comment by erictapen** [comment]
Thanks, renamed that category in https://github.com/erictapen/mapping-migration/commit/57f80fef5adec2f662f135a917d4a23373577d55!

**Comment by erictapen** [comment]
As a mockup for the presentation I'll implement an unclickable button `ⓘ` next to every decision category.

**Comment by herbertblb** [comment]
see #18

**Comment by seragarez** [comment]
Explanation for "otherwise closed" by Leo: 
People whose asylum application was closed without a substantive decision (thus, neither recognized nor rejected). Such reasons may include inadmissibility, withdrawal, abandonment, death, Dublin II procedure (Europe only), etc.

**Comment by erictapen** [comment]
I've looked a bit into info boxes and I think I have an idea how to implement them, but it will take some effort as I have to restructure the code for it.

For starters it's already easily possible to write a titletext for the ⓘ icons. It's displayed on mouse over and currently reads "coming soon".

**Comment by erictapen** [comment]
And again I posted to the wrong issue ;) Closed, see https://github.com/erictapen/mapping-migration/issues/38#issuecomment-1048833332 !
